index.html

```diff
@@ -53,6 +53,169 @@ <h1>
+                        <li>JavaScript</li>
+                        <li>HTML</li>
+                    </ul>
+                    <button>See Project</button>
+                </div>
+            </div>
+            <div class="card">
+                <div class="card_image">
+                    <img src="logo/Img Plaholder.png" alt="Image placeholder">
+                </div>
+                <div class="card_content">
+                    <h3>Multi-Post Stories <br> Gain+Glory</h3>
+                    <ul>
+                        <li>Ruby on Rails</li>
+                        <li>CSS</li>
+                        <li>JavaScript</li>
+                        <li>HTML</li>
+                    </ul>
+                    <button>See Project</button>
+                </div>
+            </div>
+        </div>
+    </section>
+    <!-- About -->
+    <section class="about">
+        <div class="about_content">
+            <div>
+                <div class="illustration1">
+                    <h3>About Me</h3>
+                    <img src="logo/illustration 1- about me-mobile.png" alt="">
+                </div>
+                <p> I can help you build a product , feature or website Look through some of my work and experience! If
+                    you like what you see and have a project you need coded,
+                    don’t hesitate to contact me.</p>
+                <button>Get my Resume</button>
+            </div>
+            <div class="about_boxes">
+                <div class="box">
+                    <div> <img src="logo/icon-languages.png" alt="Languages"></div>
+                    <h3>Languages</h3>
+                    <ul>
+                        <li>JavaScript</li>
+                        <li>Ruby</li>
+                        <li>HTML</li>
+                        <li>CSS</li>
+                    </ul>
+                </div>
+                <div class="box">
+                    <div> <img src="logo/icon-frameworks.png" alt="Frameworks"></div>
+                    <h3>Frameworks</h3>
+                    <ul>
+                        <li>React.js</li>
+                        <li>Ruby on Rails</li>
+                        <li>RSpec</li>
+                    </ul>
+                    <ul>
+                        <li>Capybara</li>
+                        <li>Selenium</li>
+                    </ul>
+                </div>
+                <div class="box">
+                    <div> <img src="logo/icon-skills.png" alt="Skils"></div>
+                    <h3>Skills</h3>
+                    <ul>
+                        <li>Database Management</li>
+                        <li>Version Control</li>
+
+                    </ul>
+                    <ul>
+                        <li>CLI</li>
+                        <li>Web Development</li>
+                        <li>API Design</li>
+                    </ul>
+                </div>
+            </div>
+        </div>
+
+    </section>
```

Comment:
Great UI! 👍 However, there is a small issue with the vertical scrolling on the page. As per the design, there should be no vertical scrollbar visible. To fix this, you need to ensure that the content on the page fits within the viewport without requiring scrolling. 

Please review the content, layout, and sizing of the elements to ensure they are properly adjusted to eliminate the vertical scrolling. By addressing this issue, you will achieve a more polished and seamless user experience. Keep up the good work! 💯 

[Check this video](https://github.com/Cheptoo21/Portfolio-setup-and-mobile-first/assets/55840999/7ee47607-bf80-4e48-be85-4e96dc45bb8a)